Xero.NetStandard.OAuth2/Client/ApiException.cs

```diff
@@ -33,13 +33,13 @@ public class ApiException : Exception
         /// Gets or sets Retry-After value
         /// </summary>
         /// <value>The Retry-After (Http response header Retry-After).</value>
-        public int? RetryAfter { get; set; }
+        public int? RetryAfter { get; private set; }
 
         /// <summary>
         /// Gets or sets the Limit Type value
         /// </summary>
         /// <value>The Limit Type (Http response header X-Rate-Limit-Problem).</value>
-        public string LimitType { get; set; }
+        public string LimitType { get; private set; }
```

Comment:
As above regarding private. Also as above regarding the possible values of this variable included in the xml comment